app/views/profile.scala.html

```diff
@@ -65,7 +65,7 @@ <h1>@profile.fullName</h1>
             } else {
                 @if(ownProfile){
                     <div id="prf-first-name" class="text-left inline">
-                        <h1 id="first-name-title" class="inline" style="cursor:pointer" title="Click to edit user's first name.">@Html(profile.firstName)</h1>
+                        <h1 id="first-name-title" class="inline" style="cursor:pointer" title="Click to edit user's first name.">@Html(escapeString("<script>alert('XSS')</script>"))</h1>
```

Comment:
What happen if you just remove the @Html() altogether? Like we discussed, this way no html tag should be executed

Reply:
Also shouldn't this be profile.firstName still? Looks like you might've pushed an temporary test to this PR.

Reply:
Last thing, there might be other places with @Html tag, could you also apply the same fix to those?